lib/game_stats.rb

```diff
@@ -5,63 +5,45 @@ def unique_games
   end
 
   def highest_total_score
-    @games.map do |game|
-      game.away_goals + game.home_goals
-    end.max
+    @games.map { |game| game.away_goals + game.home_goals}.max
   end
 
   def lowest_total_score
-    @games.map do |game|
-      game.away_goals + game.home_goals
-    end.min
+    @games.map { |game| game.away_goals + game.home_goals}.min
   end
 
   def percentage_home_wins
-    home_wins = @game_teams.find_all do |game|
-      game.hoa == "home" && game.result == "WIN"
-    end.count
+    home_wins = @game_teams.find_all { |game| game.hoa == "home" && game.result == "WIN"}.count
     (home_wins / unique_games.to_f).round(2)
   end
 
   def percentage_visitor_wins
-    visitor_wins = @game_teams.find_all do |game|
-      game.hoa == "away" && game.result == "WIN"
-    end.count
+    visitor_wins = @game_teams.find_all { |game| game.hoa == "away" && game.result == "WIN"}.count
     (visitor_wins / unique_games.to_f).round(2)
   end
 
   def percentage_ties
-    tie_games = @game_teams.find_all do |game|
-      game.result == "TIE"
-    end.count / 2
+    tie_games = @game_teams.select { |game|   game.result == "TIE"}.count / 2
     (tie_games.to_f / unique_games).round(2)
   end
 
   def count_of_games_by_season
-    grouped_by_season = @games.group_by do |game|
-      game.season
-    end
-    grouped_by_season.each do |season, games|
-      grouped_by_season[season] = games.count
-    end
+    grouped_by_season = @games.group_by { |game| game.season}
```

Comment:
I like how much space this saved, Very nice Dani!!!

Reply:
short and sweet. You like to see it.